paddle/fluid/operators/matmul_op.h

```diff
@@ -72,23 +73,57 @@ Tensor CombineBatchAndM(const Tensor& input) {
 // (Warning: This requires transposing data and writes into new memory.)
 // Identity op if the tensor is not of rank 3.
 template <typename DeviceContext, typename T>
-Tensor CombineBatchAndN(const DeviceContext& context, const Tensor& input) {
-  Tensor output;
+inline framework::Tensor CombineBatchAndN(const DeviceContext& context,
```

Comment:
CombineBatchAndN => UnfoldLastTwoDims

Reply:
Rename it to `FoldHeadLastDims`. It will combine the 1st and last dimension together.